h2o-genmodel/src/main/java/hex/genmodel/CategoricalEncoding.java

```diff
@@ -72,8 +72,24 @@
     }
   };
 
+  private final boolean _parametrized;
+
+  CategoricalEncoding(boolean parametrized) {
+    _parametrized = parametrized;
+  }
+
   public abstract Map<String, Integer> createColumnMapping(GenModel m);
 
   public abstract Map<Integer, CategoricalEncoder> createCategoricalEncoders(GenModel m, Map<String, Integer> columnMapping);
-  
+
+  /**
+   * Does the categorical encoding have any parameters that are needed to correctly interpret it?
+   * Eg.: number of classes for EnumLimited
+   * 
+   * @return Is this encoding parametrized?
+   */
+  public boolean isParametrized() {
+    return _parametrized;
+  }
+
```

Comment:
I was hoping you would also start to extend this interface to make it easier to convert highly-parametrized TE into an implementation of `CategoricalEncoding` ;)

Reply:
:)